LabBuilder/LabBuilder.psm1

```diff
@@ -2774,7 +2889,8 @@ Add-Content ``
     $SetupCompleteCmd = @"
 @echo SetupComplete.cmd Script Started... >> %SYSTEMROOT%\Setup\Scripts\SetupComplete.log
 $SetupCompleteCmd
-powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`"
+Timeout 30
+powerShell.exe -ExecutionPolicy Unrestricted -Command `"%SYSTEMROOT%\Setup\Scripts\SetupComplete.ps1`" `r
```

Comment:
I'm still puzzled by this one (I'm having some possibly related problems with encoding of JSON for the unit tests), but we'll leave it in and revisit later if necessary.